paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerConfigurationHandler.kt

```diff
@@ -23,55 +24,66 @@ internal class FlowControllerConfigurationHandler @Inject constructor(
     private val paymentSelectionUpdater: PaymentSelectionUpdater,
 ) {
 
-    suspend fun configure(
+    private var job: Job? = null
+
+    fun configure(
+        scope: CoroutineScope,
+        initializationMode: PaymentSheet.InitializationMode,
+        configuration: PaymentSheet.Configuration?,
+        callback: PaymentSheet.FlowController.ConfigCallback,
+    ) {
+        job?.cancel()
```

Comment:
I'm wondering if we should use atomicreference for this. That way we can use `getAndSet`, as well as the thread safe access.

Reply:
I actually just added this in https://github.com/stripe/stripe-android/pull/6488/commits/313151519ca9c6e8d09f8fa698f0cf2368cfe6ae. Happy to pull it into this pull request though!

Reply:
I think it makes sense to pull in here, and use `getAndSet` rather than get and set separately. As is in the other PR, you could still race where you might not cancel!